drivers/lora/Kconfig.sx12xx

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) 2019 Manivannan Sadhasivam
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+DT_COMPAT_SEMTECH_SX1276 := semtech,sx1276
+
+menuconfig LORA_SX12XX
+	bool "Semtech SX-series driver"
+	select HAS_SEMTECH_RADIO_DRIVERS
+	select HAS_SEMTECH_LORAMAC
+	depends on SPI
+	help
+	  Enable LoRa driver for Semtech SX12xx.
+
+if LORA_SX12XX
+
+choice
+	prompt "LoRa Radio chipset"
+	default LORA_SX1276 if $(dt_compat_enabled,$(DT_COMPAT_SEMTECH_SX1276))
+	help
+	  Antenna connection type.
```

Comment:
This description is wrong, choice is about "LoRa Radio chipset".

Reply:
Ack